HW03/anastasia_evsihneyeva/anastasia_evsihneyeva_hw_03_t_01.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class Homework3
+  def task1(logs)
+    variable = logs.to_s.split("\n")
```

Comment:
Should work without `.to_s`
```suggestion
    variable = logs.split("\n")
```